planck/PLKClojureScriptEngine.m

```diff
@@ -551,7 +557,86 @@ -(void)startInitializationWithSrcPaths:(NSArray*)srcPaths outPath:(NSString*)out
                                             name:@"PLANCK_IS_DIRECTORY"
                                          argList:@"path"
                                        inContext:self.context];
-        
+
+        [ABYUtils installGlobalFunctionWithBlock:
+         ^JSValueRef(JSContextRef ctx, size_t argc, const JSValueRef argv[]) {
+             
+             if (argc == 1 && JSValueGetType (ctx, argv[0]) == kJSTypeString) {
+
+                 NSString* path = NSStringFromJSValueRef(ctx, argv[0]);
+                 
+                 if (![[NSFileManager defaultManager] fileExistsAtPath:path]) {
+                     return JSValueMakeNull(ctx);
+                 }
+
+                 NSDictionary* result = [[NSFileManager defaultManager]attributesOfItemAtPath:path error:nil];
+                 
+                 bool appendOnly = [ result objectForKey:NSFileAppendOnly];
+                 bool fileBusy = [ result objectForKey:NSFileBusy];
+                 double deviceId = [[ result objectForKey:NSFileDeviceIdentifier] doubleValue];
+                 bool extensionHidden = [ result objectForKey:NSFileExtensionHidden];
+                 double creatorCode = [[result objectForKey:NSFileHFSCreatorCode] doubleValue];
+                 double typeCode = [[result objectForKey:NSFileHFSTypeCode] doubleValue];
+                 bool immutable = [result objectForKey:NSFileImmutable];
```

Comment:
When I run this code, I'm the result dictionary only has 15 key/value pairs, but 18 are being pulled out here. One example missing one is `NSFileImmutable`.

Perhaps this code should be revised so that only the key/value pairs that exist in result are returned in the ClojureScript map.